src/tsconfig.json

```diff
@@ -0,0 +1,31 @@
+{
+  "compilerOptions": {
+    "strict": false,
+    "strictNullChecks": false,
+    "noUnusedLocals": false,
+    "noUnusedParameters": false,
+    "noImplicitReturns": false,
+    "noImplicitAny": false,
+    "noImplicitThis": false,
+    "noFallthroughCasesInSwitch": false,
+    "declaration": true,
+    "emitDecoratorMetadata": true,
+    "experimentalDecorators": true,
```

Comment:
I don't believe this project uses decorators (please correct me if I'm wrong).

Reply:
I use this tsconfig template everywhere, so I don't have to be worried about changing it for every single project. But I can remove the unused parts.

Reply:
Fixed all the tsconfig comments in this: https://github.com/atom/atom-select-list/pull/28/commits/4b5e63b73cce3f3ec560dfe1467a816dd20f93d9